account_analytic_analysis_recurring/account_analytic_analysis_recurring.py

```diff
@@ -243,8 +243,10 @@ def recurring_create_invoice(self, cr, uid, automatic=False, context=None):
                 new_date = next_date + relativedelta(days=+interval)
             elif contract.recurring_rule_type == 'weekly':
                 new_date = next_date + relativedelta(weeks=+interval)
-            else:
+            elif contract.recurring_rule_type == 'monthly':
                 new_date = next_date + relativedelta(months=+interval)
+            else:
+                new_date = next_date+relativedelta(years=+interval)
```

Comment:
Add spaces around the `+`. 
The elif and else blocks look identical. Why do we need two code blocks then?